test/composition.cancel.js

```diff
@@ -28,12 +28,14 @@ metatests.test('async functions composition cancel in the middle', test => {
   };
 
   const fn3 = (data, cb) => {
+    test.fail('canceled function called');
     wrapAsync(() => {
```

Comment:
I don't think these wrapAsync calls are needed here as they are not supposed to be called anyway.